woocommerce/woocommerce-android
# Shipping Labels: Add "activate service packages" form.

# What this PR does

This PR adds a functionality to allow users to activate a service package (also known as predefined package), during shipping label creation process.

Fixes #3992

# Testing Instructions
0. Make sure the test site has Shipping Labels creation enabled, and have an Order where Shipping Labels can be created.
1. Delete all activated packages in wp-admin: `wp-admin/admin.php?page=wc-settings&tab=shipping&section=woocommerce-services-settings`.
2. In the app, find the Order from step 0 and create a Shipping Label, complete the "Ship from" and "Ship to" steps, then continue with the "Packaging details" step.
3. Tap the "Package selected" spinner, this will open the "Add new package" screen. 
4. Switch to the "Service Package" tab. This will show a loading screen, and then finally show a list of active packages that can be activated.
5. Select a package (remember the category and name), tap Done.
6. If the package is saved properly,  the screen will be navigated back to the "Package Details" screen. Make sure the name of the recently activated package is shown in the "Package selected" spinner.
7. Tap the "Package selected" spinner. Make sure this opens the "Selected Package" screen, and the recently added package is listed there.
8. Tap the "Add New Package" button again. Open the "Service Package" tab and make sure that the recently activated package is no longer listed there.
9. Pick another package there, and tap "Done". This will again navigate back to the "Package Details" screen. Make sure the package name is now shown in the "Package selected" spinner.

See video below for the entire flow:

https://user-images.githubusercontent.com/266376/126443151-cce449f6-3147-4495-922d-a3d2d497886a.mp4



# Known limitations
- No unit tests yet. This will be added in a separate PR.

Update release notes:
